MainClass.java

```diff
@@ -0,0 +1,25 @@
+package FirstPackage;

```

Comment:
Comment chung:
- Làm đúng yêu cầu, có thêm chức năng nhập ký tự từ bàn phím --> good job
- Tuy nhiên về trình bày code vẫn lộn xộn: code không thẳng hàng. Để chỉnh lại format có thể dùng tổ hợp phím bôi đen toàn bộ code: Ctrl+A --> Ctrl+I